test/index.js

```diff
@@ -19,19 +19,25 @@ test('validate-npm-package-name', function (t) {
 
   // Scoped (npm 2+)
 
-  t.deepEqual(validate('@npm/thingy'), {validForNewPackages: true, validForOldPackages: true})
+  t.deepEqual(validate('@npm/thingy'), {
```

Comment:
Scoped public packages can start with @ so isn't this testcase incorrect ?

Reply:
Added a message in the issue and tagged Stefan to confirm this.

Reply:
Corrected it. @ should be allowed